Makefile

```diff
@@ -32,8 +32,6 @@ build-debug:
 package:
 	cd build-opt; \
 	mv dragonfly $(RELEASE_NAME); \
-	tar cvfz $(RELEASE_NAME).unstripped.tar.gz $(RELEASE_NAME) ../LICENSE.md; \
-	strip $(RELEASE_NAME); \
```

Comment:
If we don't strip this here? Do we need fall into the same problem of https://github.com/dragonflydb/dragonfly/issues/999? 🤔

Reply:
The build flags change is doing two things - it reduces the 'level' of the debug symbols from 2 to 1, and it tells gcc to compress them. So we get a binary with debug symbols and a reasonable size of 16MB which we can use unstripped.

Reply:
Ah Nice!